File: backend/brand-service/src/main/java/vou/com/example/brand/entity/Brand.java
package vou.com.example.brand.entity;

import jakarta.persistence.*;
import lombok.Data;

import java.util.List;

@Entity
@Table(name = "brand")
@Data
public class Brand {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private long id;

    @Column(name = "username", unique = true)
    private String username;

    @Column(name = "password")
    private String password;

    @Column(name = "category")
    private String category;

    @Column(name = "address")
    private String address;

    @Column(name = "lon")
    private String lon;

    @Column(name = "lat")
    private String lat;
  
    @Column(name = "state")
    private boolean state = false;
}

File: backend/brand-management-service/internal/model/brand.go
package model

type Brand struct {
	ID       int16  `gorm:"primary_key" json:"id"`
	Username string `json:"username,omitempty"`
	Password string `json:"password,omitempty"`
	Category string `json:"category,omitempty"`
	Address  string `json:"address,omitempty"`
	Lon      string `json:"lon,omitempty"`
	Lat      string `json:"lat,omitempty"`
	State    bool   `json:"state"`
}

type UpdateBrandRequest struct {
	Category string `json:"category,omitempty"`
	Address  string `json:"address,omitempty"`
	Lon      string `json:"lon,omitempty"`
	Lat      string `json:"lat,omitempty"`
}
